Makefile

```diff
@@ -317,6 +317,11 @@ test-yaml:
 	@# Fall back to find if there is no git, e.g. in package builds
 	yamllint --strict $$((git ls-files "*.yml" "*.yaml" 2>/dev/null || find -name '*.y*ml') | grep -v ^dbicdh)
 
+.PHONY: test-shfmt
+test-shfmt:
```

Comment:
you should add this in https://github.com/os-autoinst/openQA/pull/5440/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R296 as well, otherwise those checks won't be called ever again

Reply:
I wrote several times in this PR that we cannot **yet** use it because it is not installed in the container.

Reply:
oh, I thought that would have been handled already.

Reply:
I created
* https://github.com/os-autoinst/os-autoinst/pull/2449
* https://github.com/os-autoinst/openQA/pull/5444

to add shfmt to dependencies first.